lib/inc/sairedis.h

```diff
@@ -205,4 +205,15 @@ typedef enum _sai_redis_switch_attr_t
      */
     SAI_REDIS_SWITCH_ATTR_RECORDING_FILENAME,
 
+    /**
+     * @brief Get operation response timeout in milliseconds.
+     *
+     * Also used for every synchronous API call.
+     *
+     * @type sai_uint64_t
+     * @flgs CREATE_AND_SET
+     * @default 60000
+     */
+    SAI_REDIS_SWITCH_ATTR_GET_RESPONSE_TIMEOUT_MS,
```

Comment:
I believe it should be SET_RESPONSE_TIMEOUT_MS

Reply:
mainly this timeout was designed for GET api in any quad api, but with adding synchronous mode it's used for all of them